### DISCUSSION TITLE
Can backup schedules only be defined via the CLI tool?

### DISCUSSION THREAD

**Discussion by jdheyburn** [vmware-tanzu/velero#3946] [Community support Q&A] [answered]
I am aware that the `velero` CLI tool is required to define backups - but is this the only way?

Is it possible to define backup schedules with a CRD, so that the user does not have to install the tool just establish backups?

Thanks!

**Comment by ghost** [comment, marked as answer]
If you take a look at the helm chart; you can see where it's defined in the [values file](https://github.com/vmware-tanzu/helm-charts/blob/main/charts/velero/values.yaml#L350-L365).

You can deconstruct that and see that it just creates a resource using the CRD [here](https://github.com/vmware-tanzu/helm-charts/blob/main/charts/velero/templates/schedule.yaml)
